src/components/Lesson.vue

```diff
@@ -79,14 +94,14 @@
             </div>
           </div>
           <div class="lh-copy pv2 ph3" v-else>
-            Update the code to complete the exercise. Click <strong>submit</strong> to check your answer.
+          Update the code to complete the exercise. Click <strong>submit</strong> to check your answer.
           </div>
         </div>
         <div class="pt3 ph2 tr">
-          <div v-if="output.test && output.test.success && lessonNumber === lessonsInWorkshop">
+          <div v-if="((output.test && output.test.success) || lessonPassed) && lessonNumber === lessonsInWorkshop">
             <Button v-bind:click="workshopMenu" class="bg-aqua white">More Workshops</Button>
           </div>
-          <div v-else-if="output.test && output.test.success">
+          <div v-else-if="(output.test && output.test.success) && lessonPassed || lessonPassed">
```

Comment:
you might want to put one more set of parens in here cause it's not clear to me how the conditions are being paired.

Reply:
Good call. Was actually able to simplify this one.